binstar_client/utils/projects/models.py

```diff
@@ -56,10 +56,16 @@ def configuration(self):
         output = self.metadata.get('configuration', {})
         output.update({
             'size': self.size,
-            'num_of_files': len(self.pfiles)
+            'num_of_files': len(self)
         })
         return output
 
+    def __len__(self):
```

Comment:
This might be better as a normal (non-magic) method like `get_file_count`. 

`__len__` is great for containers, but in this case, we don't support iteration, containment testing, etc. Also, conceptually, a project contains multiple entity types such as files, environments, services, variables, and commands... better to be explicit.